toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/SceneView.kt

```diff
@@ -211,8 +211,41 @@ public fun SceneView(
         content = content
     )
 }
+
 /**
- * A compose equivalent of the view-based [SceneView].
+ * A composable UI element that displays three-dimensional (3D) geographic content defined by an [ArcGISScene].
+ *
+ * A composable scene view displays [ArcGISScene] layers and graphics in 3D. It uses a [Camera] to control
+ * the visible area (extent) of the [ArcGISScene] (see [cameraController]) and supports user interactions such as pan,
+ * zoom, tilt, and rotate.
+ * It also provides access to the underlying layer data in the scene.
+ *
+ * User interactions (pan, zoom, tilt, rotate, identify, selection) are supported using touch, mouse, keyboard,
+ * or pen/pencil input on the composable. If required, you can override keys, clicks, and gestures to provide
+ * a specific user experience by using callbacks such as:
+ * - [onPan]
+ * - [onSingleTapConfirmed]
+ * - [onDoubleTap]
+ * - etc.
+ *
+ * The visible area ([Viewpoint]) of the composable scene view is defined by the position and orientation of a [Camera].
+ * The current visible area can be accessed through the lambda callbacks:
+ * - [onViewpointChangedForCenterAndScale]
+ * - [onViewpointChangedForBoundingGeometry]
+ * - [onCurrentViewpointCameraChanged]
+ *
+ * These callbacks are triggered when navigation completes or the viewpoint changes.
+ *
+ * To programmatically set the viewpoint, use a [SceneViewProxy] passed to the composable and call methods such as
+ * [SceneViewProxy.setViewpointAnimated] or [SceneViewProxy.setViewpointCamera].
+ *
+ * In an MVC architecture, this composable represents the View tier. The Model tier is the [ArcGISScene], which can
+ * provide operational layers, base map, and a base surface.
```

Comment:
I don't know if MVC is very common for composable UIs. Better to change this to MVVM, since it also has a model and a view and compose developers will likely know about ViewModels.
```suggestion
 * In an MVVM architecture, this composable represents the View tier. The Model tier is the [ArcGISScene], which can
 * provide operational layers, base map, and a base surface.
```